mne/viz/topo.py

```diff
@@ -331,8 +332,15 @@ def _imshow_tfr(ax, ch_idx, tmin, tmax, vmin, vmax, onselect, ylim=None,
     # construct a time-frequency bounds grid
     time_mesh, freq_mesh = np.meshgrid(time_lims, freq_lims)
 
-    img = ax.pcolormesh(time_mesh, freq_mesh, tfr[ch_idx], cmap=cmap,
-                        vmin=vmin, vmax=vmax)
+    if mask is not None:
```

Comment:
We need input checks here ... but we should probably wait for my evoked mask PR, which has a bunch of input checks already, and then do a combined function in a new PR.